### DISCUSSION TITLE
Is togglecompass coming back?

### DISCUSSION THREAD

**Discussion by kellyhamlin** [Zrips/CMI#6980] [Q&A]
I noticed it was quietly removed a couple releases back and config options had changed. You can now use it with compass in hand or recovery compass for death location. Just curious what drove that, perhaps I missed the change notes. I have more players that like it than not, so overall not a problem I suppose. Just want to better understand the preferred new use case here.

Thanks

**Comment by CloudeLecaw** [comment]
U can set this in Options also by command :)
```

cmi options (playerName) (option) (enable/disable/toggle/status) (-s)
Modify personal options
```

You see can create for all in Options a little custom alais if needed :)

**Comment by mrfloris** [comment]
Bit late, but, if you wish to re-add the old commands you can consider using my little add-on that maps the old to new commands; https://github.com/mrfdev/CMI/tree/master/Resources/Add-ons/legacytoggles
